indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -63,7 +66,15 @@
 def _render_registration_details(registration):
     event = registration.registration_form.event
     tpl = get_template_module('events/registration/management/_registration_details.html')
-    return tpl.render_registration_details(registration=registration, payment_enabled=event.has_feature('payment'))
+
+    assigned_tags = RegistrationTagSchema(many=True).dump(registration.tags)
+    assigned_tags.sort(key=lambda tag: str.casefold(tag['name']))
+
+    all_tags = RegistrationTagSchema(many=True).dump(RegistrationTag.query.with_parent(event).all())
```

Comment:
Since you use it twice it'd be better to instantiate it just once: `schema = RegistrationTagSchema(many=True)`